osclib/checkrepo.py

```diff
@@ -439,6 +439,12 @@ def check_specs(self, request_id=None, request=None):
         specs = [en.attrib['name'][:-5] for en in root.findall('entry')
                  if en.attrib['name'].endswith('.spec')]
 
+        # special case for glibc.i686, it have not the relevant specfile for glibc.i686
+        # but must be add it to requests list as a dummy request, otherwise the state
+        # has not be check and won't download it's binaries.
+        if 'glibc' in specs:
```

Comment:
uhm TBH I do not understand the situation. There are three spec files in glibc (glibc, glibc-utils and glibc-testsuite), and you are adding a new one (glibc.i686). For this last one there will be a request to OBS to get the md5, some _link test and more. If this spec file is not there, there is a chance that the request will be declined.

Did you tested it?